pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+> Open source java application designed for load testing functional behavior and measure performance.
```

Comment:
I have a suggestion should this line be 
```suggestion
> Open source java application designed for load testing functional behavior and measuring performance.
```
 or 
```suggestion
> Open source java application designed for load testing functional behavior and to measure performance.
 ```
 I think the current one feels a bit odd with *measure perforamance* alone. What do you guys think @navarroaxel @CleanMachine1 @EmilySeville7cfg ?

Reply:
I prefer leave as-is.